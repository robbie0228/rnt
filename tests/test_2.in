board
move a up
move a left
move a right
board
move A down
board
move a down
board
move A down
move A left
move A right
board
move a up
board
move A up
board
move a down
board
move A up
board
move a down
board
move A up
board
move a down
board
move B up
board
quit
