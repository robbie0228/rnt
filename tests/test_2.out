Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Cannot move link off this border!

Cannot move link off this border!

Cannot move a link onto another of your links!

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Invalid link!

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Cannot move link off this border!

Cannot move link off this border!

Cannot move a link onto another of your links!

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Invalid link!

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
a..de...
........
........
........
........
A..DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
...de...
a.......
........
........
........
A..DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
...de...
a.......
........
........
A.......
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
...de...
........
a.......
........
A.......
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
...de...
........
a.......
A.......
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 1V
Abilities: 5
a: V1   b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
...de...
........
........
a.......
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 1:
Downloaded: 0D, 1V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
...de...
........
........
a.......
........
.B.DE...
..CSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

